dev/TreeView/TreeView.idl

```diff
@@ -94,6 +94,14 @@ runtimeclass TreeViewDragItemsCompletedEventArgs
     }
 }
 
+[WUXC_VERSION_RS4]
+[webhosthidden]
+runtimeclass TreeViewSelectionChangedEventArgs
```

Comment:
I am not sure if this should go trough API review, @ranjeshj

Reply:
Yes, this need API approval before we can add it.  @jeremy.alles@gmail.com can you please add the tag to this type? there are examples in some of the other idl files, progress ring for sure

---
In reply to: [391084955](https://github.com/microsoft/microsoft-ui-xaml/pull/2063#discussion_r391084955) [](ancestors = 391084955)

Reply:
Hmm actually, I'm not sure to do this. If I replace `[WUXC_VERSION_RS4]` by `[WUXC_VERSION_PREVIEW]` I'm getting a compiler error because that doesn't match what is set at the `TreeView` level.

Reply:
I think you also need to mark the event handler you are introducing with the [WUXC_VERSION_PREVIEW] attribute, as it is using that type.

Reply:
Got it, done now. Thanks.